File: ESC.Resturant.Data/Entities/Unit.cs
using ESC.Resturant.Data.Entities;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace ESC.Resturant.Data.Entities
{
    public class Unit
    {
        [Key]
        public int Id { get; set; }
        public string NameAr { get; set; }
        public string NameEn { get; set; }
        public bool? IsForSell { get; set; }
        public bool? IsForBuy { get; set; }
        public bool IsDelete { get; set; }
        public virtual ICollection<Item> Item { get; set; }
        public virtual ICollection<BillDetails> BillDetails { get; set; }


    }
}

File: ESAResturant.NEW.web/ClientApp/src/app/admin/item/Models/Unit.ts
import { Item } from "./Item";
import { BillDetail } from "../../bill-invoice/Models/BillDetail";

export class Unit {

  constructor() {
    this.id = 0;
    this.nameAr = '';
    this.nameEn = '';
    this.isForSell = false;
    this.isForBuy = false;
    this.isDelete = false;
    this.item=[]
    this.billDetails = [];
  }


  id: number;
  nameAr: string;
  nameEn: string;
  isForSell: boolean;
  isForBuy: boolean;
  isDelete: boolean;
  item: Item[];
  billDetails: BillDetail[]
}
